util/launcher/src/migrate.rs

```diff
@@ -29,6 +30,7 @@ impl Migrate {
         migrations.add_migration(Box::new(migrations::AddChainRootMMR)); // TODO(light-client) update the comment: which version?
         migrations.add_migration(Box::new(migrations::AddBlockFilterColumnFamily)); // since v0.105.0
         migrations.add_migration(Box::new(migrations::AddBlockFilterHash)); // since v0.108.0
+        migrations.add_migration(Box::new(migrations::BlockExt2019ToZero::new(hardforks))); // since v0.111.1
```

Comment:
Should we add a "Release Note" about the affect about `BlockExt2019ToZero` migration?

Reply:
This will cause ckb to need to manually perform magration, and the release note should have prompts.